package.json

```diff
@@ -80,16 +81,17 @@
     "mocha": "^3.5.3",
     "nyc": "^14.1.1",
     "prettier": "^1.18.2",
-    "rollup": "^1.4.0",
-    "rollup-plugin-filesize": "^6.0.1",
-    "rollup-plugin-replace": "^2.1.0",
-    "rollup-plugin-terser": "^4.0.4",
-    "rollup-plugin-typescript2": "^0.21.1",
-    "rollup-plugin-uglify": "^6.0.0",
+    "rollup": "^1.19.4",
+    "rollup-plugin-filesize": "^6.2.0",
+    "rollup-plugin-replace": "^2.2.0",
+    "rollup-plugin-terser": "^5.1.1",
+    "rollup-plugin-typescript2": "^0.22.1",
+    "rollup-plugin-uglify": "^6.0.2",
```

Comment:
Is `rollup-plugin-uglify` actually used? I don't see any reference to it in `rollup.config.js` 🤔

Reply:
No, its not - I’ve actually just upgraded all rollup-related packages and ive taken them from package.json. Would you like to maybe prepare a PR with removing this?